gordo_components/builder/mlflow_utils.py

```diff
@@ -262,11 +262,20 @@ def get_batch_kwargs(metadata: dict) -> dict:
     model_keys = ["model-creation-date", "model-builder-version", "model-offset"]
     param_list += get_params(build_metadata["model"], model_keys)
 
+    # Parse cross-validation split metadata
+    try:
+        splits = build_metadata["model"]["cross-validation"]["splits"]
+        param_list += get_params(splits, splits.keys())
```

Comment:
minor detail, but if the `try` block is ensuring that the previous line doesn't error, then the second should be in the `else` section. If this line, for whatever reason in the future also raises a `KeyError` then the logged error is not actually relevant.

Reply:
Fixed here and for the others :+1: